tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+                <option value="TAP" selected="true">TAP: Tables</option>
+                <option value="SCS">SCS: Cone search</option>
+                <option value="SSA">SSA: Spectra</option>
+                <option value="SIA">SIA: Images</option>
+              </param>
+              <param name="wavebands" type="select" label="Wavebands">
+                <option value="all" selected="true">All</option>
+                <option value="radio">Radio</option>
+                <option value="millimeter">Millimeter</option>
+                <option value="infrared">Infrared</option>
+                <option value="optical">Optical</option>
+                <option value="uv">UV</option>
+                <option value="euv">EUV</option>
+                <option value="x-ray">X-ray</option>
+                <option value="gamma-ray">Gamma-ray</option>
+              </param>
+            </when>
+            <when value="archive">
+              <param name="archive" type="select" label="Archive">
+                <options from_data_table="astronomical_archives" />
+              </param>
+            </when>
+        </conditional>
+        <section name="query_section" title="Query selection" expanded="false">
+          <conditional name="query_selection">
+            <param name="query_type" type="select" label="ADQL Query Selection">
+              <option value="none">No specific query (first n files from archive)</option>
+              <option value="obscore_query">IVOA obscore table query builder</option>
+              <option value="raw_query">Raw ADQL query builder</option>
+            </param>
+            <when value="none"></when>
+            <when value="obscore_query">
+              <!-- Fields defined https://www.ivoa.net/documents/ObsCore/20160224/WD-ObsCore-v1.1-20160224.pdf -->
+              <param name="target_name" type="text" label="Observation target name" help="Typically name of the astronomical object observed" />
+              <param name="t_min" type="float" optional="true" label="Start time in MJD" />
+              <param name="t_max" type="float" optional="true" label="Stop time in MJD" />
+              <!-- Dataproduct_type field values definition https://www.ivoa.net/rdf/product-type/2023-06-26/product-type.html  -->
+              <conditional name="dataproduct_type_selection">
+                <param name="dataproduct_input_type" type="select" label="Data product type input choice">
+                  <option value="list" selected="true">From list</option>
+                  <option value="custom">Custom (Input data product type yourself)</option>
+                </param>
+                <when value="list">
+                  <param name="dataproduct_type_select" type="select" label="Data product type" help="Logical data product type (link to reference in Help section)">
+                    <option value="none" selected="true">None</option>
```

Comment:
This None things we can omit by simply add `optional="true"` to the select.

Reply:
What value will it have when it is optional?

Reply:
A pyhon "None" I think.